authorizer-app/src/app/admin/services/user.service.ts

```diff
@@ -68,4 +69,28 @@ export class UserService {
     );
     return this.http.delete(url);
   }
+
+  storeUserAuthParams(url: string) {
+    const params = this.getJsonFromUrl(url);
+    localStorage.setItem(
+      this.AUTH_ENDPOINT_PARAMS_STORAGE_KEY,
```

Comment:
Maybe add the key to enums in `StorageItem` in `"@app/shared/enums/storage-item"` so it can be accessed from outside the class.